node/src/actors/messages.rs

```diff
@@ -192,6 +192,8 @@ pub struct BuildVtt {
     pub vto: Vec<ValueTransferOutput>,
     /// Fee
     pub fee: u64,
+    /// weighted fee flag
+    pub weighted_fee: bool,
```

Comment:
Similar to the CLI, I would prefer the JSON-RPC API to accept either `"fee":1` or `"weighted_fee":1`, if that's possible.

Reply:
it would be a breking change then